libcontainer/specconv/spec_linux.go

```diff
@@ -818,6 +823,33 @@ func parseMountOptions(options []string) (int, []int, string, int) {
 		"rslave":      unix.MS_SLAVE | unix.MS_REC,
 		"runbindable": unix.MS_UNBINDABLE | unix.MS_REC,
 	}
+
+	// See https://man7.org/linux/man-pages/man2/mount_setattr.2.html
+	recAttrFlags := map[string]struct {
```

Comment:
Unrelated, but if we call `parseMountOptions` a few times, it's better to implement lazy initOnce for these maps and make them global.

Reply:
Implemented as part of https://github.com/opencontainers/runc/pull/3281

Reply:
Sorry, I mean, it can easily be implemented on top of #3281.

Reply:
Thanks, updated